1

```diff
@@ -0,0 +1,13 @@
+Disable test on CoreCLR
+
+The CompareAllBytesEmitted tests need to be disabled on CoreCLR due to
```

Comment:
Is addition of this file intentional?

Reply:
No. Not sure how that got there. Thanks for catching.

Reply:
Ah it was a bad vim save 😦